storage/engine/mvcc.go

```diff
@@ -480,11 +480,12 @@ func mvccGetInternal(engine Engine, key proto.Key, metaKey proto.EncodedKey, tim
 	}
 	// If we're doing inconsistent reads and there's an intent, we
 	// ignore the intent by insisting that the timestamp we're reading
-	// at is an historical timestamp < the intent timestamp.
-	if !consistent && meta.Txn != nil {
-		if !timestamp.Less(meta.Timestamp) {
-			timestamp = meta.Timestamp.Prev()
-		}
+	// at is a historical timestamp < the intent timestamp. However, we
```

Comment:
so the caller will get a value and a WriteIntentError, and it basically needs to know that because it was reading inconsistently, the value is the legitimate result of the inconsistent read?